### PR TITLE
postgrest backend small improvements

### PR DESCRIPTION
**Pull request by demianfe**

### CODE REVIEW

File: ormin/ormin_postgre.nim [hunk 1]
```diff
@@ -95,12 +98,17 @@ template bindResult*(db: DbConn; s: PStmt; idx: int; dest: bool;
...
-  while src[i] != '\0':
-    dest.add src[i]
-    inc i
-    
+  var
+    c = src[0]
+    i = 0
+  while c != '\0':
+    dest.add($c)
+    i = i + 1
```
Comment (Araq) [inline ormin/ormin_postgre.nim]: what? I think the old ``fillString`` was fine.

File: tools/ormin_importer.nim [hunk 2]
```diff
@@ -148,11 +148,65 @@ proc attrToKey(a: DbColumn; t: KnownTables): int =
...
+
+
+proc replaceType(line: string): string =
+  result = line
+  for alias in pgAliases:
+    if line.contains alias[0]:
+      result = result.replace(alias[0], alias[1])
+  
+
+proc parsePgFile(strm: FileStream, infile: string): SqlNode =
```
Comment (Araq) [inline tools/ormin_importer.nim]: Can't we fix the SQL parser instead?

Comment (demianfe) [reply tools/ormin_importer.nim]: Ok, I'll give it a shot next week.

File: ormin/ormin_postgre.nim (hunk 1 above)
Comment (demianfe) [reply ormin/ormin_postgre.nim]: To be honest I don't remember anymore as it was a while ago, but if is I did it, it was for a reason. I'll provide more info tomorrow.

Comment (demianfe) [reply ormin/ormin_postgre.nim]: Confirmed, old fillString works fine now, I think is due to some changes in Nim itself. And I see that some other issues that i had previously are also working now with Nim 1.0.0. So I'll create Issues with the specific findings and then PRs with the fixes, so feel free to reject this PR if that's OK with you.

**Comment by Araq** [comment]
Closing as suggested by @demianfe.

**Pull request closed without merging**